.github/scripts/generate_binary_build_matrix.py

```diff
@@ -146,22 +146,22 @@ def arch_type(arch_version: str) -> str:
 
 
 # This can be updated to the release version when cutting release branch, i.e. 2.1
-DEFAULT_TAG = os.getenv("RELEASE_VERSION_TAG", "main")
+DEFAULT_TAG = "efc34a54fb31461a8b1aac0e95602d7785f9df20"
 
 WHEEL_CONTAINER_IMAGES = {
-    "11.8": f"pytorch/manylinux-builder:cuda11.8-{DEFAULT_TAG}",
```

Comment:
Since this PR is not modifying Docker image. We don't need this change to ecr. I believe Docker image for Rocm already available on dockerhub:
```
docker pull pytorch/manylinux2_28-builder:rocm6.2.4-main
```